examples/reference/panes/SVG.ipynb

```diff
@@ -21,7 +21,8 @@
     "\n",
     "For layout and styling related parameters see the [customization user guide](../../user_guide/Customization.ipynb).\n",
     "\n",
-    "* **``embed``** (boolean, default=False): If given a URL to an image this determines whether the image will be embedded as base64 or merely linked to.\n",
+    "* **``embed``** (boole, default=False): If given a URL to an image this determines whether the image will be embedded as base64 or merely linked to.\n",
+    "* **``encode``** (bool, default=True): Whether to base64 encode the SVG, when enabled SVG links may not work.\n",
```

Comment:
Why is it true by default if it lacks linking? Presumably encoding has some benefit, but what it is is not stated.

Reply:
Just didn't change the default but I guess there's no compelling reason not to, base64 is slightly more efficient I guess but probably not worth it if certain features don't work.

Reply:
Ok, I vote to change the default then.